operations/meeting-notes.rst

```diff
@@ -20,6 +20,41 @@ Dne 18. 11. Zuzana Javorková požádala o grant do výše 10 000 Kč na tisk tr
 
 Grant byl schválen.
 
+23. 10. 2019 - pyvec board schuze
```

Comment:
Já vím, že jsem nudnej, ale napsal bych spíš "schůze výboru". (Kdyby se tě už tolik lidí jako mě ptalo, jaký je rozdíl mezi boardem, radou a výborem, tak seš taky nudnej :D )

Reply:
za me "schuze vyboru" super. jdu se mrknout na #120 protoze Petr to vytunil!